src/main/java/org/jitsi/videobridge/rest/LongPollingServlet.java

```diff
@@ -38,7 +59,55 @@ protected void doGet(
         throws IOException,
                ServletException
     {
-        // TODO Auto-generated method stub
-        response.setStatus(HttpServletResponse.SC_NOT_FOUND);
+        System.out.println("BB: IN LONG POLLING");
+        Continuation continuation = ContinuationSupport.getContinuation(request);
```

Comment:
The last bit of information on the subject of asynchronous processing of HTTP requests that we exchanged via e-mail was:

We’ll use asynchronous processing of HTTP requests defined by the Servlet 3.0 specification. Please refer to https://docs.oracle.com/javaee/6/api/javax/servlet/ServletRequest.html#startAsync(javax.servlet.ServletRequest, javax.servlet.ServletResponse).

I know I mentioned Jetty Continuations in person prior to that e-mail exchange when we were still in the support investigation phase. I'd like us to go with the Servlet 3.0 API because we have it already in our dependency list (while Jetty Continuations is yet another dependency) and is equally easy to work with.